salt/modules/test.py

```diff
@@ -20,7 +20,7 @@
 
 __proxyenabled__ = ['*']
 
-
+exit(1)
```

Comment:
This will just fill up the logs, can we make a module in the integration test suite that does this instead?